src/pages/cart/CartPage.jsx

```diff
@@ -1,9 +1,22 @@
-import React from "react"
+import React from "react";
+import Navbar from "../../components/shared/navbar/NavbarComponent";
+import Footer from "../../components/shared/footer/FooterComponent";
+import CartProducts from "../../components/cart/CartProducts";
+import CartCheckout from "../../components/cart/CartCheckout";
 
-const CartPage = ()=>{
-    return(
-        <h1>This is Cart Page</h1>
-    )
-}
+const CartPage = () => {
+  return (
+    <>
+      <div className="cart-pg-layout">
+        <div className="child cart-pg-navbar">
+          <Navbar />
+        </div>
```

Comment:
why have you placed the navbar here?

Reply:
as they have separate grid styles...will try to refactor and have only once